requirements_dev.txt

```diff
@@ -46,4 +46,6 @@ tensorflow==1.13.2
 protobuf>=3.6.0
 vcrpy>=2.0.0
 pytest-vcr>=1.0.0
+torch
+torchvision
```

Comment:
CI installs these using a custom pytorch repo: https://github.com/wandb/client/blob/master/.circleci/config.yml#L34 You should just add a message to the readme for devs.